index.js

```diff
@@ -1,6 +1,20 @@
 'use strict';
 const os = require('os');
-const home = os.homedir();
+const isWindows = require('is-windows');
+
+const home = (() => {
+	const homeDir = os.homedir();
+	if (isWindows()) {
+		// Return the home directory with the user name part in 8.3 format
+		// (https://goo.gl/FxW2CB) if needed. This is a workaround for
+		// https://github.com/nodejs/node/issues/17586.
+		const tmpDir = os.tmpdir();
+		const userDir = homeDir.replace(/([^\\])+$/g, '');
```

Comment:
This will be something like `C:\\Users\\`.

(Note that the disk drive `C` and the `Users` part can differ depending on the system settings and language.)